h/static/scripts/directives/markdown.coffee

```diff
@@ -336,7 +336,7 @@ markdown = ['$filter', '$sanitize', '$sce', '$timeout', ($filter, $sanitize, $sc
     # React to the changes to the input
     inputEl.bind 'blur change keyup', ->
       ctrl.$setViewValue inputEl.val()
-      scope.$digest()
+#      scope.$digest()  # TODO: this shouldn't be a problem
```

Comment:
While debugging this *digest*ion problem, you might want to comment out [this line](https://github.com/hypothesis/h/blob/export-showframe/h/static/scripts/guest.coffee#L386), to reduce the number of things going on simultaneously in Angular.